gh-93096: Make `mimetypes` CLI tool public (#93097)
python/cpython: Misc/NEWS.d/next/Library/2022-05-28-19-41-02.gh-issue-93096.qjUyRG.rst

> Command-line :mod:`mimetypes` started to return 1 instead of 0 for
> abruptions and 2 instead of 1 for incorrect command line parameters.

Reviewer: abruptions? Perhaps:
```suggestion
Command-line :mod:`mimetypes` started to exit with 1 instead of 0
on failure and 2 instead of 1 for incorrect command line parameters.
```

> Also, errors are printed into stderr instead of stdout.

Reviewer: ```suggestion
Also, errors are printed to stderr instead of stdout.
```
